test/TradeTrustERC721.js

```diff
@@ -65,4 +69,84 @@ contract("TradeTrustErc721", accounts => {
       /VM Exception while processing transaction: revert ERC721: transfer caller is not owner nor approved/
     );
   });
+
+  describe("Surrendered TradeTrustERC721 Work Flow", () => {
+    let tokenRegistryInstanceWithShippingLineWallet;
+    let tokenRegistryAddress;
+    let escrowInstance;
+    let escrowInstanceAddress;
+    let escrowInstance1;
+    let escrowInstanceAddress1;
+
+    beforeEach("", async () => {
+      // Setup in accordance to workflow, starting test after the point of surrendering ERC721 Token
+      tokenRegistryInstanceWithShippingLineWallet = await Erc721.new("foo", "bar", { from: shippingLine });
+      tokenRegistryAddress = tokenRegistryInstanceWithShippingLineWallet.address;
+      escrowInstance = await TitleEscrow.new(tokenRegistryAddress, beneficiary1, beneficiary1, ZERO_ADDRESS, {
+        from: beneficiary1,
+      });
+      escrowInstanceAddress = escrowInstance.address;
+      escrowInstance1 = await TitleEscrow.new(tokenRegistryAddress, beneficiary1, beneficiary1, ZERO_ADDRESS, {
+        from: beneficiary1,
+      });
+      escrowInstanceAddress1 = escrowInstance1.address;
+      await tokenRegistryInstanceWithShippingLineWallet.safeMint(escrowInstanceAddress, merkleRoot);
+      await escrowInstance.transferTo(tokenRegistryAddress, {
+        from: beneficiary1,
+      });
+    });
+
+    it("should be able to destroy token", async () => {
```

Comment:
What is this really testing for?

Reply:
hmm do you mean the test is wrong or the description is misleading?

Reply:
Ah sorry misread the code.